app/Http/Controllers/UserController.php

```diff
@@ -83,10 +83,9 @@ public function index()
 
     public function friend_profile($id)
     {
-        $user = User::where('id',$id)->first();
-        $me = Auth::id();
-        $me=User::where('id',$id)->first();
-        return view('friend_profile', ['user' => $user,'me'=>$me]);
+        $user = User::find($id);
+        $is_friend = Friend::where('user_id',Auth::id())->where('friend_id',$id)->get();
```

Comment:
`$is_xxxx` のような変数はbooleanというtrue/falseのいずれかの値が入ることを意味する事が多いです。
そして今回はデータがあるかどうかを検証したいだけなので下記にように `exists` を使うと良いと思います
```suggestion
        $is_friend = Friend::where('user_id',Auth::id())->where('friend_id',$id)->exists();
```
https://readouble.com/laravel/6.x/ja/queries.html
（レコード存在の判定）